hack/e2e-kind-config.yaml

```diff
@@ -1,15 +1,29 @@
 # this config file contains all config fields with comments
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
+featureGates:
+  DynamicResourceAllocation: true
+  DRAResourceClaimDeviceStatus: true
+containerdConfigPatches:
+  # Enable CDI as described in
+  # https://tags.cncf.io/container-device-interface#containerd-configuration
+  - |-
+    [plugins."io.containerd.grpc.v1.cri"]
```

Comment:
So we need also enable this when using DRA?

Reply:
Yes, mounting DRA device needs to use CDI